synapse/lib/storm.py

```diff
@@ -1784,15 +1784,15 @@ def dmonWarn(evnt):
 
                 self.status = 'running'
                 async with await self.core.snap(user=self.user, view=view) as snap:
-                    snap.cachebuids = False
-
                     snap.on('warn', dmonWarn)
                     snap.on('print', dmonPrint)
                     self.err_evnt.clear()
 
                     async for nodepath in snap.storm(text, opts=opts, user=self.user):
                         # all storm tasks yield often to prevent latency
                         self.count += 1
+                        snap.buidcache.clear()
+                        snap.livenodes.clear()
```

Comment:
This only works when a dmon yields nodes.

Reply:
Updated to do this differently